src/components/SideBar.js

```diff
@@ -159,36 +201,47 @@ const SideBar = () => {
   ];
 
   return (
-    <Stack
-      css={`
-        width: 230px;
-        background-color: ${getValueForSideBar('backgroundColor')};
-        height: 100vh;
-        color: ${getValueForSideBar('color')};
-        z-index: 2000;
-      `}
-      padding={2}
-    >
-      <Link href="/dashboard">
-        <Logo />
-        {/* <Logo variants={LogoVariants.MOBILE} /> */}
-      </Link>
+    <>
       <Stack
-        spacing={4}
         css={`
-          flex: 1;
-          > :not(:first-child) {
-            border-top: 1px solid ${getValueForMenu('borderColor')};
-          }
+          width: 230px;
+          background-color: ${getValueForSideBar('backgroundColor')};
+          height: 100vh;
+          color: ${getValueForSideBar('color')};
+          z-index: 1998;
         `}
+        padding={2}
       >
-        <Menu items={userMenu} />
-        <Stack justifyContent="space-between" css="flex: 1;">
-          <Menu items={manageMenu} title={t('menu.management')} />
-          <Menu items={notificationMenu} />
+        <Link href="/dashboard">
+          <Logo />
+          {/* <Logo variants={LogoVariants.MOBILE} /> */}
+        </Link>
+        <Stack
+          spacing={4}
+          css={`
+            flex: 1;
+            > :not(:first-child) {
+              border-top: 1px solid ${getValueForMenu('borderColor')};
+            }
+          `}
+        >
+          <Menu items={userMenu} />
+          <Stack justifyContent="space-between" css="flex: 1;">
+            <Menu items={manageMenu} title={t('menu.management')} />
+            <Menu items={notificationMenu} />
+          </Stack>
         </Stack>
       </Stack>
-    </Stack>
+      <AnnouncementsModal
+        visible={isModalVisible}
+        announcements={announcements}
+        seenAnnouncements={seenAnnouncements}
```

Comment:
Can't we fix this by manipulating the data? Changing the status of the announcement and taking that into consideration while rendering?